integration_test.go

```diff
@@ -320,3 +326,27 @@ func TestDownload(t *testing.T) {
 		t.Fatal(err)
 	}
 }
+
+func TestGoroutineLeak(t *testing.T) {
```

Comment:
How do you check goroutine leak?

Reply:
If goroutine leak, test will not end because session stuck.

So, routine integration test will fail with test timeout.

If not leak, test end normally.

I think it is not enough so add final check in this commit. (https://github.com/pyama86/pftp/pull/33/commits/54da412037bf9086331a9d01206ff7ad236ad569)

This test will compare goroutine counts before and after load test.

If after test goroutine counts is bigger, goroutine has a leak so test fail.

Reply:
It looks like good solution!
It seems to be a sure-fire way to count the number of goroutines.